tests/transformers/test_transformers_inference_utils.py

```diff
@@ -0,0 +1,51 @@
+from mlflow.transformers.inference_utils import (
+    _get_finish_reason,
+    _get_output_and_usage_from_tensor,
+    _get_token_usage,
+)
+
+
+def test_output_dict_for_completions(text_generation_pipeline):
```

Comment:
nit: Can we mock tokenizer to simple `lambda s: s.spilt(" ")` to make tests faster and simpler? Transformers test is super slow already.....

Reply:
Got it, I just mocked all the return values of the tokenizer functions in the `test_transformers_llm_inference_utils.py` file

Reply:
It still loads `text_generation_pipeline` fixture which takes time. I think you can do
```
class DummyTokenizer:
     input_ids = torch.tensor([1, 2, 3])

     def decode(self, s, ...):
        return s.split(" ")
...

pipeline = mock.MagicMock()
pipeline.tokenizer.return_value = DummyTokenizer()
```

Reply:
Makes sense, thanks for pointing this out! Indeed the fixture loads models which is not efficient nor necessary in this case.